---
layout: cnv-page
classes: wide
toc: true
title: "22q11.2 Copy Number Variation Syndrome"
permalink: /22q11-2/
author_profile: false
cnv: "22q11.2"
---

<!-- CNV description fetched from the ORPHADATA via the API with granted access-->
### Description
<p>ORPHAnet: Get it if available from ORPHADATA via their API when given access</p>

### Chromosomal location <!-- TODO -->
{% assign chr = site.data.cnv-data[page.cnv].chromosome %}
{% assign start = site.data.cnv-data[page.cnv].start %}
{% assign end = site.data.cnv-data[page.cnv].end %}

<p>Chr{{ chr }}:{{ start }}-{{ end }}</p>

<!-- TODO Sequence Viewer -->
</p> Add image like the genome viewer <!-- TODO --> </p>

### Prevalence <!-- TODO -->
<p>Get it if available from ORPHADATA via their API when given access</p>

### Common phenotypic features <!-- TODO -->
<p>{{ site.data.cnv-data[page.cnv].phenotypic_features }}</p>

### Detailed clinical description <!-- TODO -->
<p>Get it if available from ORPHADATA via their API when given access</p>

<!-- WikiPathways Viewer-->
### Interactive Pathway Viewer
<!-- In the pathway_id = site.data['wikipathways-ids']['cnv'] change the cnv with the name of the CNV that is already stored in the _data/wikipathways-ids.yml -->
{% assign pathway_id = site.data.cnv-data[page.cnv].wikipathways_id %}
{% include wikipathway-viewer.html id=pathway_id %}

<!-- Get the Gene table from _includes/gene-table.html
There we have the structure of the table to which we can add dinamically the rows based on the given geneSymbols list -->
# Gene Information 
{% include gene-table.html %} <!-- from _includes/gene-table.html -->

<!-- Call the function to add to the Gene table the information fetched from HGNC and from ORPHADATA -->
<script type="module">
    import { displayGeneLinks } from '/assets/js/populate-gene-table.js';

    const geneSymbols = {% assign gene_list = site.data.cnv-data[page.cnv].genes %}{{ gene_list | jsonify }};
    displayGeneLinks(geneSymbols);
</script>

<!-- TODO: Fix the references so they can be linked in the text -->
### References
<p>
[1]: Rapini, Ronald P.; Bolognia, Jean L.; Jorizzo, Joseph L. (2007). *Dermatology: 2-Volume Set*. St. Louis: Mosby. ISBN 978-1-4160-2999-1. <br> 
[2] Zamariolli M, Auwerx C, Sadler MC, van der Graaf A, Lepik K, Schoeler T, Moysés-Oliveira M, Dantas AG, Melaragno MI, Kutalik Z. The impact of 22q11. 2 copy-number variants on human traits in the general population. The American Journal of Human Genetics. 2023 Feb 2;110(2):300-13 <br>
[3]: James, William D.; Berger, Timothy G.; et al. (2006). *Andrews' Diseases of the Skin: Clinical Dermatology*. Saunders Elsevier. ISBN 978-0-7216-2921-6. <br>
[4]: "22q11.2 deletion syndrome". Genetic and Rare Diseases Information Center (GARD). Archived from the original on 5 July 2017. Retrieved 15 May 2017. <br>
[5]: Shprintzen RJ, Goldberg RB, Lewin ML, Sidoti EJ, Berkman MD, Argamaso RV, Young D (January 1978). "A new syndrome involving cleft palate, cardiac anomalies, typical facies, and learning disabilities: velo-cardio-facial syndrome". *Cleft Palate J.* 15 (1): 56–62. PMID 272242.</p>